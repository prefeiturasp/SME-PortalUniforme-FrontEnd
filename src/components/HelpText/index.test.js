import { mount } from "enzyme";
import React from "react";
import { HelpText } from "./index";

describe("test <HelpText>", () => {
  it("renders component", () => {
    let wrapper = mount(<HelpText helpText="Teste" />);

    expect(wrapper).toMatchSnapshot();
  });
});
