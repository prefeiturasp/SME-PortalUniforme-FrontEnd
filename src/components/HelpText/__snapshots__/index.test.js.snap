// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <HelpText> renders component 1`] = `
<HelpText
  helpText="Teste"
>
  <div
    className="help-text"
  >
    Teste
  </div>
</HelpText>
`;
